src/acceleration/ConstantRelaxationAcceleration.cpp

```diff
@@ -30,17 +30,8 @@ void ConstantRelaxationAcceleration::initialize(DataMap &cplData)
 {
   checkDataIDs(cplData);
 
-  // Append column for old values if not done by coupling scheme yet
-  int entries = 0;
-  for (auto &elem : _dataIDs) {
-    entries += cplData[elem]->values().size();
-  }
-  for (DataMap::value_type &pair : cplData) {
-    int cols = pair.second->oldValues.cols();
-    if (cols < 1) {
-      PRECICE_ASSERT(pair.second->values().size() > 0, pair.first);
-      utils::append(pair.second->oldValues, (Eigen::VectorXd) Eigen::VectorXd::Zero(pair.second->values().size()));
-    }
+  for (DataMap::value_type &pair : cplData) { // @todo: seems to be unnecessary, since this is again just a loop over CouplingScheme::getAccelerationData
```

Comment:
see above

Reply:
Solved in 4046df9b6a92019d634b952f4ff8e1bc8523940a.